app/lib/forms/sign_in.rb

```diff
@@ -30,5 +32,9 @@ def after_save
         ConfirmEmailMailer.confirmation_code_mail(to: email, code:).deliver_now
       end
     end
+
+    def skip_sending_otp
```

Comment:
Cosmetic: `skip_sending_otp?` or maybe `omit_otp?`. Besides, maybe instead of doing it on per step basis, it could be left as it is and on development otp could accept static code like `12345`. But its of course out of scope for and leave it as topic for discussion - especially I am not familiar with this functionality yet.

Reply:
100% this needs some love and could be added to the backlog for changes!

Reply:
to be honest in dev or even testing i dont see why its used; static number actually better your right though! might include